terraform/demo/main.tf

```diff
@@ -109,4 +109,9 @@ module "gro_ingestion_service" {
 
   environment = local.env
   region      = data.aws_region.current.name
+
+  gdx_url                 = module.data_share_service.gdx_url
+  auth_url                = module.data_share_service.token_auth_url
+  publisher_client_id     = module.data_share_service.gro_ingestion_client_id
+  publisher_client_secret = module.data_share_service.gro_ingestion_client_secret
```

Comment:
Is this the raw credential passing around/will it end up tfstate? If so, is there any sensible way we can use SSM/inject the secret without knowing it as much?

Reply:
It will end up in tfstate, I can mark it as sensitive, but this is the same as the current setup for our consumer.